packages/subproviders/src/types.ts

```diff
@@ -10,16 +10,14 @@ export interface LedgerCommunicationClient {
  * NodeJs and Browser communication are supported.
  */
 export interface LedgerEthereumClient {
-    // shouldGetChainCode is defined as `true` instead of `boolean` because other types rely on the assumption
-    // that we get back the chain code and we don't have dependent types to express it properly
```

Comment:
Can we please keep the type of `shouldGetChainCode` as `true`? Any reason you changed this?

Reply:
I didn't even see that tricky little true! Not intentional